app/core/Tracker2/LegacyTracker.js

```diff
@@ -0,0 +1,53 @@
+import CocoLegacyTracker from '../Tracker'
+import BaseTracker from './BaseTracker'
+
+/**
+ * Acts as a proxy between our new tracker and our legacy Tracker.coffee
+ * which still handles segment.io and GA tracking.  This tracker also binds
+ * application events to the legacy tracker and exposes legacy tracker methods
+ * externally as necessary.
+ *
+ * TODO remove this tracker when tracker refactor is complete.
+ */
+export default class LegacyTracker extends BaseTracker {
+  constructor (store, cookieConsent) {
+    super()
+
+    this.store = store
+    this.cookieConsent = cookieConsent
+  }
+
+  async _initializeTracker () {
```

Comment:
Should this method block on anything before resolving the promise?

Reply:
Don't think so because legacy tracker doesn't have any async init processes that are exposed but let me know if you're seeing something I'm not!